src/lib/helpers.test.ts

```diff
@@ -518,34 +531,35 @@ const allNSCapabilities: CapabilityExport[] = JSON.parse(`[
   }
 ]`);
 
-const nonNsViolation: CapabilityExport[] = JSON.parse(`[
+const nonNsViolation: CapabilityExport[] = [
```

Comment:
We're passing an actual object here instead of a brittle string that doesn't keep current with any changes to type definitions. I wonder why it was a big string in the first place.